python/tvm/autotvm/measure/measure_methods.py

```diff
@@ -479,9 +479,10 @@ def run_through_rpc(measure_input, build_result,
         if ref_input:
             args = [nd.array(x, ctx=ctx) for x in ref_input]
         else:
-            # create empty arrays on the remote device and copy them once.
+            # create random arrays on the remote device and copy them once.
             # This can avoid some memory issues that make the measurement results unreliable.
-            args = [nd.empty(x[0], dtype=x[1], ctx=ctx) for x in build_result.arg_info]
+            args = [nd.array(np.random.uniform(0.0, 255.0, size=x[0]).astype(dtype=x[1]), ctx=ctx)
```

Comment:
This will introduce a data copy when using RPCRunner, which will bring too much overhead.
One way to solve this is implementing a `tvm.nd.non_empty` or `tvm.nd.random` in the tvm runtime, then we can do the random fill on the target device without copying over network

Reply:
@merrymercy Sure. I will port it to our upstream soon.

Reply:
I open https://github.com/apache/incubator-tvm/issues/5216 to track this